docker-setup.sh

```
@@ -0,0 +1,130 @@
+        exit 1
+    fi
+else
+    echo ".env file already exists"
+fi
+
+# Generate BETTER_AUTH_SECRET if not set
+if ! grep -q "BETTER_AUTH_SECRET=.*[^=]$" .env 2>/dev/null || grep -q "BETTER_AUTH_SECRET=your-better-auth-secret" .env 2>/dev/null; then
+    echo "Generating BETTER_AUTH_SECRET..."
+    SECRET=$(openssl rand -base64 32)
+    # Escape special characters for sed
+    ESCAPED_SECRET=$(printf '%s\n' "$SECRET" | sed 's/[[\.*^$()+?{|]/\\&/g')
```

Shellcheck (0.11.0)

[warning] ESCAPED_SECRET appears unused. Verify use (or export if used externally).

(SC2034)